node/core/provisioner/src/lib.rs

```diff
@@ -324,7 +324,23 @@ fn note_provisionable_data(
 				.with_para_id(backed_candidate.descriptor().para_id);
 			per_relay_parent.backed_candidates.push(backed_candidate)
 		},
-		_ => {},
+		// We choose to do nothing with misbehavior at this stage
```

Comment:
We should argue why not here as well. The misbehavior is self-contained and the validator could get punished. The reason we don't is, because we already have mitigation on the protocol level (reputation changes) and for the time being those seem to be enough.